.github/FUNDING.yml

```
@@ -0,0 +1 @@
+custom: ["https://numfocus.org/donate-to-napari"]
```

```suggestion
github: numfocus
custom: http://numfocus.org/donate-to-napari
```

looking around at other numfocus sponsored projects, I see that they all have `github: numfocus`, (described on the [github sponsors](https://docs.github.com/en/enterprise-cloud@latest/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/displaying-a-sponsor-button-in-your-repository#about-funding-files) bit of the funding docs) ... see for example, [xarray](https://github.com/pydata/xarray) so let's do that! :)